lib/Doctrine/Migrations/Provider/LazySchemaDiffProvider.php

```diff
@@ -18,59 +14,27 @@
  */
 class LazySchemaDiffProvider implements SchemaDiffProvider
 {
-    private LazyLoadingValueHolderFactory $proxyFactory;
-
     private SchemaDiffProvider $originalSchemaManipulator;
 
     public function __construct(
-        LazyLoadingValueHolderFactory $proxyFactory,
         SchemaDiffProvider $originalSchemaManipulator
     ) {
-        $this->proxyFactory              = $proxyFactory;
         $this->originalSchemaManipulator = $originalSchemaManipulator;
     }
 
-    public static function fromDefaultProxyFactoryConfiguration(
```

Comment:
this method should probably be deprecated instead of being removed

Reply:
the class is marked `@internal` so I didn't bother doing that :)

Reply:
ah indeed. I missed that